app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt

```diff
@@ -106,7 +106,9 @@ fun BonusSectionView(
       Image(
         modifier = Modifier
           .padding(top = 58.dp)
-          .minimumInteractiveComponentSize(),
+          .minimumInteractiveComponentSize()
+          .size(24.dp)
+          .padding(start = 3.dp, end = 3.dp, bottom = 3.dp),
```

Comment:
In this case it might be easier to just set the size to 18.dp and have no paddings